test/services/role/services/userPermissions.test.js

```diff
@@ -0,0 +1,94 @@
+const { expect } = require('chai');
+const app = require('../../../../src/app');
+const testObjects = require('../../helpers/testObjects')(app);
+
+const userPermissions = app.service('/permissions/user');
+
+describe('userPermissions', async () => {
+	const ROLES = {
+		TEST: 'test',
+		OTHER: 'other',
+		EXTENDED: 'extended',
+		MULTIPLE: 'multiple',
+		XX: 'xx',
+		NOTHING: 'nothing spezial',
+  };
+  const testPermissions = [
+		'SINGING',
+		'DANCE_RAIN',
+		'WALK_LINES',
+		'RUN_FLOOR',
+  ];
+  
+  	const otherPermissions = [
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5786177)